contrib/builder/rpm/amd64/photon-1.0RC/Dockerfile

```diff
@@ -0,0 +1,18 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/amd64/generate.sh"!
+#
+
+FROM photon:1.0RC
```

Comment:
`1.0RC` is going to be supported until GA comes out (sometimes later this year), once GA is out we can decide to either keep RC artifacts around for another 6 months or just drop them in favor of GA.